src/hotspot/share/nmt/memTracker.hpp

```diff
@@ -262,8 +262,8 @@ class MemTracker : AllStatic {
   static void tuning_statistics(outputStream* out);
 
   // MallocLimt: Given an allocation size s, check if mallocing this much
-  // under category f would hit either the global limit or the limit for category f.
-  static inline bool check_exceeds_limit(size_t s, MEMFLAGS f);
+  // under category f would hit either the global limit or the limit for mem_tag.
```

Comment:
I don't know what "category f" is.   Maybe reword as
    // MallocLimit: Given an allocation size s, check if allocating this much memory would hit the global limit or the
   // limit tagged with mem_tag.

Reply:
Fixed.